src/Features/CSharp/Portable/BraceCompletion/AbstractCurlyBraceOrBracketCompletionService.cs

```diff
@@ -48,7 +48,6 @@ internal abstract class AbstractCurlyBraceOrBracketCompletionService : AbstractC
 
             var (formattingChanges, finalCurlyBraceEnd) = FormatTrackingSpan(
                 context.Document,
-                context.Document.LanguageServices.ProjectServices,
```

Comment:
unnecessarily redundant with the ParsedDocument being passed in.  removed this parameter.